Use dump-as-markup.js in editing/inserting/insert-at-end-01.html and insert-at-end-02.html

rniwa opened the pull request:
#### f9d5fdae696e1d85a4f4c75890a5168034c4713a
<pre>
Use dump-as-markup.js in editing/inserting/insert-at-end-01.html and insert-at-end-02.html
<a href="https://bugs.webkit.org/show_bug.cgi?id=252608">https://bugs.webkit.org/show_bug.cgi?id=252608</a>

Reviewed by Wenson Hsieh.

Use dump-as-markup.js in these tests for clarity.

* LayoutTests/editing/inserting/insert-at-end-01-expected.txt: Added.
* LayoutTests/editing/inserting/insert-at-end-01.html:
* LayoutTests/editing/inserting/insert-at-end-02-expected.txt: Added.
* LayoutTests/editing/inserting/insert-at-end-02.html:
* LayoutTests/platform/gtk/editing/inserting/insert-at-end-01-expected.png: Removed.
* LayoutTests/platform/gtk/editing/inserting/insert-at-end-01-expected.txt: Removed.
* LayoutTests/platform/gtk/editing/inserting/insert-at-end-02-expected.png: Removed.
* LayoutTests/platform/gtk/editing/inserting/insert-at-end-02-expected.txt: Removed.
* LayoutTests/platform/ios-wk2/editing/inserting/insert-at-end-01-expected.txt: Removed.
* LayoutTests/platform/ios-wk2/editing/inserting/insert-at-end-02-expected.txt: Removed.
* LayoutTests/platform/mac/editing/inserting/insert-at-end-01-expected.png: Removed.
* LayoutTests/platform/mac/editing/inserting/insert-at-end-01-expected.txt: Removed.
* LayoutTests/platform/mac/editing/inserting/insert-at-end-02-expected.png: Removed.
* LayoutTests/platform/mac/editing/inserting/insert-at-end-02-expected.txt: Removed.
* LayoutTests/platform/wincairo/editing/inserting/insert-at-end-01-expected.txt: Removed.
* LayoutTests/platform/wincairo/editing/inserting/insert-at-end-02-expected.txt: Removed.

Canonical link: <a href="https://commits.webkit.org/260593@main">https://commits.webkit.org/260593@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/01f00a6adb9068d97738283eb94f0a8b7b61b4b6

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/108732 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/17835 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/41581 "Built successfully") | [❌ 🛠 wpe](https://ews-build.webkit.org/#/builders/87/builds/18 "Updated wpe dependencies (failure)") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/118050 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/19284 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/9110 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/101022 "Built successfully") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/114500 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/14470 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/97696 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/35/builds/42496 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/96444 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/29334 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/34/builds/84336 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/10631 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/30685 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/11393 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/7609 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/16780 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/50275 "Passed tests") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/12979 "Built successfully") | | | 
| [❌ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/3985 "Failed to push commit to Webkit repository") | | | | 
<!--EWS-Status-Bubble-End-->

whsieh reviewed (commented):
Hm...it looks like there aren't any changes to `editing/inserting/insert-at-end-01.html` in this commit — only removed platform expectations (or maybe I missed it?)

rniwa committed:
Use dump-as-markup.js in editing/inserting/insert-at-end-01.html and insert-at-end-02.html
https://bugs.webkit.org/show_bug.cgi?id=252608

Reviewed by Wenson Hsieh.

Use dump-as-markup.js in these tests for clarity.

* LayoutTests/editing/inserting/insert-at-end-01-expected.txt: Added.
* LayoutTests/editing/inserting/insert-at-end-01.html:
* LayoutTests/editing/inserting/insert-at-end-02-expected.txt: Added.
* LayoutTests/editing/inserting/insert-at-end-02.html:
* LayoutTests/platform/gtk/editing/inserting/insert-at-end-01-expected.png: Removed.
* LayoutTests/platform/gtk/editing/inserting/insert-at-end-01-expected.txt: Removed.
* LayoutTests/platform/gtk/editing/inserting/insert-at-end-02-expected.png: Removed.
* LayoutTests/platform/gtk/editing/inserting/insert-at-end-02-expected.txt: Removed.
* LayoutTests/platform/ios-wk2/editing/inserting/insert-at-end-01-expected.txt: Removed.
* LayoutTests/platform/ios-wk2/editing/inserting/insert-at-end-02-expected.txt: Removed.
* LayoutTests/platform/mac/editing/inserting/insert-at-end-01-expected.png: Removed.
* LayoutTests/platform/mac/editing/inserting/insert-at-end-01-expected.txt: Removed.
* LayoutTests/platform/mac/editing/inserting/insert-at-end-02-expected.png: Removed.
* LayoutTests/platform/mac/editing/inserting/insert-at-end-02-expected.txt: Removed.
* LayoutTests/platform/wincairo/editing/inserting/insert-at-end-01-expected.txt: Removed.
* LayoutTests/platform/wincairo/editing/inserting/insert-at-end-02-expected.txt: Removed.

Canonical link: https://commits.webkit.org/260593@main
